lab_2/main.py

```diff
@@ -34,6 +56,14 @@ def get_language_features(language_profiles: dict) -> list or None:
     :param language_profiles: a dictionary of dictionaries - language profiles
     """
     pass
+    features = []
+    if (not isinstance(language_profiles,dict)) or (len(language_profiles) == 0):
+        return None
+    for tokens_and_frequencies in language_profiles.values():
+        for token in tokens_and_frequencies:
+            features.append(token)
+            features = sorted(features)
```

Comment:
Why not sort it later when all the unique words are extracted?

Reply:
done